### ISSUE TITLE
Legacy video widget missing

### ISSUE THREAD

**Issue by rickmgithub** [Automattic/wp-calypso#55937]
### Steps to Reproduce

URL:scribblah.co.uk

PROBLEM:
* The videos in their widget sidebars do not show on the public site - https://d.pr/i/AcnRUO
* They show in the customizer but not in the published page - https://d.pr/i/k1aEKa

WHAT I FOUND:
* They are on a personal plan
* Theme is retired Bueno
* They are using a "legacy video" widget. 
* The option to add more "legacy video" widgets has vanished. I can replicate this on a test site on a personal plan and premium site. No legacy video widget exists
* The "video" widget is not an option on the personal plan

### What I expected to happen

The legacy video widget to exist as an option

### What actually happened

No new legacy video widgets can be added to test the issue and removing the existing ones could mean they lose what they did have working.

### Context

#4267605-zd

### Operating System

macOS

### Browser

Firefox

### Simple/Atomic

Simple

### Number of Users Impacted

Some users (<50%)

### Available Workarounds

Not that I can find.

### Reproducibility

Consistent

**Comment by Robertght** [comment]
I can replicate this on my end as well. I can't find it on a self-hosted website either with Jetpack active.

For now, I could recommend using the Video block as a workaround.

@Automattic/flow-patrol-create is this something that can be on your radar? 

Also, other possible related issues here: #55877 and #55696

**Comment by formosattic** [comment]
+1 4307272-zd-woothemes

Theme: Dara

The legacy widgets for Video and Gallery weren't loading on the homepage, in the sidebar. But they loaded ok in single post view.

Workaround: remove the legacy widgets, and replace with current ones that are similar, in this user's case, the YouTube block and the Slideshow block. Tried on my test site, and they loaded fine.

**Comment by arcangelini** [comment]
I attempted to test this but was not seeing the same thing on the user's live site. The videos appear to be loading properly for me in all areas. My test site showed the same thing. If you have a specific page for me to test that would be super helpful. It is also possible that this could have been resolved with the PR @Robertght mentioned Automattic/jetpack#21004

[By default the video widget is not available in the Legacy Block](https://github.com/WordPress/gutenberg/blob/ef4d395916165d7a3f89d200234aec86f4321cf7/lib/widgets.php#L61).

Given the nature of the original widget was just to embed videos from other services the user should be able to switch to the YouTube or Embed block. Slightly confusing because the Video BLOCK is not the same function as the Video WIDGET.
